mlir/test/Conversion/MathToVCIX/math-to-vcix.mlir

```diff
@@ -0,0 +1,171 @@
+// RUN: mlir-opt --split-input-file --verify-diagnostics --test-math-to-vcix %s -o - | FileCheck %s
```

Comment:
Could you add the `// -----` (exactly 5 dashes) comment between each test so that `--split-input-file` works?

I think `-o -` might not be necessary.